### DISCUSSION TITLE
[Feature] Use separate field in 'url'rather than '#' separator

### DISCUSSION THREAD

**Discussion by milnak** [ScoopInstaller/Scoop#5882] [Ideas]
#### Is your feature request related to a problem? Please describe.

Currently manifests have a way to specify a target filename for urls, e.g.

```json
    "architecture": {
        "64bit": {
            "url": "https://www.rarlab.com/rar/winrar-x64-700.exe#/dl.7z",
```

However, that's not the actual URL anymore.

#### Describe the solution you'd like

It would be nice to have a specific field for this, e.g.

```json
    "architecture": {
        "64bit": {
            "url": "https://www.rarlab.com/rar/winrar-x64-700.exe",
            "output": "dl.7z"
```

**Comment by rashil2000** [comment]
The '#' symbol is just a shorthand for making things compact - introducing another field is just added work with no apparent advantage...

What's your use case for this?

**Reply by milnak** [reply]
It's non standard and there's little saved by doing it this way, and potential problems if that url needs to be used for other reasons.

Not a blocking issue, but rather a suggestion.

**Reply by rashil2000** [reply]
I see...
